src/index.html

```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
```

Comment:
Pay attention to class names, follow the checklist 

<img width="1010" alt="Screenshot 2023-03-27 at 19 48 25" src="https://user-images.githubusercontent.com/32853034/228024541-cd76e566-9e18-47f7-9096-921c03a6dc9f.png">